core/tests/protractor_utils/workflow.js

```diff
@@ -172,6 +179,8 @@ var createAndPublishExploration = async function(
 
   var explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
   await explorationEditorPage.navigateToSettingsTab();
+  await browser.refresh();
+  await waitFor.pageToFullyLoad();
```

Comment:
Why these changes? They seem out of scope for #10798

Reply:
Actually, I used these because of the last point in #10798 i.e.

Guidelines on finding out which elements need waitFor.js function calls: Please add waitFor statements before any code that requires an element to be presented. So to understand I also take previous PR as a reference but didn't get any scope idea of scope.

Reply:
Specifically, the `browser.refresh();` seems out of scope. That changes the sequence of steps the test performs on the website, but #10798 is just about making the tests less flaky (not changing what they test). Does that distinction make sense?

Reply:
Also, I see that you already made this change. I'm leaving this thread unresolved in case you have further questions about scope